### ISSUE TITLE
List all the plugin Events Supported by Kunena

### ISSUE THREAD

**Issue by chetanmadaan** [Kunena/Kunena-Forum#6520]
Similar to:
https://docs.joomla.org/Plugin/Events

I have considering a through search but couldn't find anything. I am however happy to put a list together with sample plugins if needed be.

**Comment by 810** [comment]
its already on my todo list.

**Comment by chetanmadaan** [comment]
Do You have a draft? I can pitch in and help with some of it.

**Comment by 810** [comment]
No, i want it to be on our github wiki page.  I will create a page now.

**Comment by chetanmadaan** [comment]
Please do and I'll submit edits to it :).

**Comment by 810** [comment]
https://github.com/Kunena/Kunena-Forum/wiki/Kunena-Events

**Comment by chetanmadaan** [comment]
yaay!

**Comment by chetanmadaan** [comment]
I added the basic list and will work on documentation them one by one along with file reference as where they are called.
https://github.com/Kunena/Kunena-Forum/wiki/Kunena-Events
